securedrop/server.py

```diff
@@ -177,6 +179,19 @@ def add_contact(self, email, contact_name, contact_email):
         self.write_json()
         return ""
 
+    def list_contacts(self, email):
```

Comment:
A better name for this function is `get_contacts`, since it's just a getter.

Reply:
Changed